src/app/components/Modals/TransitionModal/StageStepping.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Button,
+  DialogBody,
+  DialogFooter,
+  DialogHeader,
+  Loader,
+  Text,
+} from "@babylonlabs-io/bbn-core-ui";
+import { twMerge } from "tailwind-merge";
+
+import { Tick } from "./Tick";
+
+const StepContent = [
```

Comment:
```suggestion
const stepContent = [
```
or we can make it uppercase